File: problems/Two Pointers/DuplicateNumbers.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        n = len(nums)
        l = 0
        r = 1
        i = 1
        if n >= 2:
            while i < n:
                if(nums[l] == nums[r]):
                    nums.pop(l)
                else:
                    l += 1
                    r += 1
                i += 1
        count = len(nums)
        return count

File: problems/Java/DuplicateNumbers.java
class Solution {
	public int removeDuplicates(int[] nums) {
		int n = nums.length;
		int l = 0;
		int r = 1;

		while (r < n) {
			if (nums[l] != nums[r] && r == l + 1) {
				r++;
				l++;
			} else if (nums[l] == nums[r]) {
				r++;
			} else {
				int temp = nums[l];
				nums[l + 1] = nums[r];
				nums[r] = temp;
				r++;
				l++;
			}
		}
		return l + 1;
	}
}